docs/source/top_level_elements/name.rst

```diff
@@ -0,0 +1,350 @@
issertant', 'Delineator', 'Dancer', 'Donor', 'Depicted', 'Depositor', 'Draftsman', 'Director', 'Designer', 'Distributor', 'Data contributor', 'Dedicatee', 'Data manager', 'Dedicator', 'Dubious author', 'Editor of compilation', 'Editor of moving image work', 'Editor', 'Engraver', 'Electrician', 'Electrotyper', 'Engineer', 'Enacting jurisdiction', 'Etcher', 'Event place', 'Expert', 'Facsimilist', 'Film distributor', 'Field director', 'Film editor', 'Film director', 'Filmmaker', 'Former owner', 'Film producer', 'Funder', 'First party', 'Forger', 'Geographic information specialist', 'Graphic technician', 'Host institution', 'Honoree', 'Host', 'Illustrator', 'Illuminator', 'Inscriber', 'Inventor', 'Issuing body', 'Instrumentalist', 'Interviewee', 'Interviewer', 'Judge', 'Jurisdiction governed', 'Laboratory', 'Librettist', 'Laboratory director', 'Lead', 'Libelee-appellee', 'Libelee', 'Lender', 'Libelee-appellant', 'Lighting designer', 'Libelant-appellee', 'Libelant', 'Libelant-appellant', 'Landscape architect', 'Licensee', 'Licensor', 'Lithographer', 'Lyricist', 'Music copyist', 'Metadata contact', 'Medium', 'Manufacture place', 'Manufacturer', 'Moderator', 'Monitor', 'Marbler', 'Markup editor', 'Musical director', 'Metal-engraver', 'Minute taker', 'Musician', 'Narrator', 'Opponent', 'Originator', 'Organizer', 'Onscreen presenter', 'Other', 'Owner', 'Panelist', 'Patron', 'Publishing director', 'Publisher', 'Project director', 'Proofreader', 'Photographer', 'Platemaker', 'Permitting agency', 'Production manager', 'Printer of plates', 'Papermaker', 'Puppeteer', 'Praeses', 'Process contact', 'Production personnel', 'Presenter', 'Performer', 'Programmer', 'Printmaker', 'Production company', 'Producer', 'Production place', 'Production designer', 'Printer', 'Provider', 'Patent applicant', 'Plaintiff-appellee', 'Plaintiff', 'Patent holder', 'Plaintiff-appellant', 'Publication place', 'Rubricator', 'Recordist', 'Recording engineer', 'Addressee', 'Radio director', 'Redaktor', 'Renderer', 'Researcher', 'Reviewer', 'Radio producer', 'Repository', 'Reporter', 'Responsible party', 'Respondent-appellee', 'Restager', 'Respondent', 'Restorationist', 'Respondent-appellant', 'Research team head', 'Research team member', 'Scientific advisor', 'Scenarist', 'Sculptor', 'Scribe', 'Sound designer', 'Secretary', 'Stage director', 'Signer', 'Supporting host', 'Seller', 'Singer', 'Speaker', 'Sponsor', 'Second party', 'Surveyor', 'Set designer', 'Setting', 'Storyteller', 'Stage manager', 'Standards body', 'Stereotyper', 'Technical director', 'Teacher', 'Thesis advisor', 'Television director', 'Television producer', 'Transcriber', 'Translator', 'Type designer', 'Typographer', 'University place', 'Voice actor', 'Videographer', 'Vocalist', 'Writer of added commentary', 'Writer of added lyrics', 'Writer of accompanying material', 'Writer of added text', 'Woodcutter', 'Wood engraver', 'Writer of introduction', 'Witness', 'Writer of preface', 'Writer of supplementary textual content']
```

Comment:
Not revelant for review here, but there are definitely roleTerms not included in our UTK MODS transform for DPLA - https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/blob/master/XSLT/utkmodstomods.xsl

Reply:
I've created an issue here:

https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/248

Reply:
Thanks!